examples/website-demos/social-messaging/src/components/SideDrawer/SideDrawer.tsx

```diff
@@ -1,15 +1,41 @@
+import { Avatar, useChatContext } from 'stream-chat-react';
+
+import { NewDirectMessage } from '../../assets/NewDirectMessage';
+import { NewGroupMessage } from '../../assets/NewGroupMessage';
+import { ThemeIcon } from '../../assets/ThemeIcon';
```

Comment:
can you export all these from an index file in assets? we're doing it that way in the virtual event app if you wanna take a look